main/cmake/APKBUILD

```diff
@@ -41,6 +41,7 @@ build() {
 		--system-curl \
 		--system-expat \
 		--no-system-jsoncpp \
+		--no-system-librhash \
```

Comment:
I’ve moved rhash to main. @pbregener, please update this PR to use system-provided rhash.